### DISCUSSION TITLE
Redirects in Caprover are 302 but should be 301

### DISCUSSION THREAD

**Discussion by darrylmorley** [caprover/caprover#2093] [Q&A] [answered]
Hi,

When redirecting for instance http://mysite.com to https://mysite.com in Caprover, the redirect is setup as a 302, not a 301. I'm aware that I can update the NGINX config to redirect with a 301, but my thinking is that it should already be a 301?

**Comment by githubsaturn** [comment, marked as answer]
302 is intentionally chosen as with 301, there is no undo.
Like what you mentioned, if your use case specifically requires 301, you can do so by modifying nginx config.
